go.mod

```diff
@@ -47,3 +49,9 @@ require (
 	k8s.io/klog/v2 v2.9.0
 	k8s.io/utils v0.0.0-20210707171843-4b05e18ac7d9
 )
+
+replace (
+	github.com/aliyun/aliyun-oss-go-sdk => github.com/menglingwei/aliyun-oss-go-sdk v1.1.13
```

Comment:
@menglingwei This is something else we should address. I do not think we should have this reference point to your fork. Is there a specific reason we are doing this replace? Can we move to the official upstream API?

Reply:
@kwoodson  ok .i submit a PR to offical upstream. When the PR is merged, i will tell you.